sdk/src/main/scala/io/horizen/AbstractSidechainApp.scala

```diff
@@ -138,6 +139,10 @@ abstract class AbstractSidechainApp
   val consensusParamsForkList = forkConfigurator.getOptionalSidechainForks.asScala.filter(fork => fork.getValue.isInstanceOf[ConsensusParamsFork])
   val defaultConsensusForks: ConsensusParamsFork = ConsensusParamsFork.DefaultConsensusParamsFork
 
+  private val forgerRewardAddress: Option[AddressProposition] = Option(sidechainSettings.forger.forgerRewardAddress)
+    .filter(_.nonEmpty)
```

Comment:
This `.nonEmpty` is for an empty string, in case the config looks like this:
```
  forger {
    restrictForgers = false
    allowedForgersList = []
    forgerRewardAddress = ""
  }
```
Otherwise it fails with an error that this is invalid address